### ISSUE TITLE
Feature Request: Clean specific rooms / zones

### ISSUE THREAD

**Issue by epheterson** [esteban-mallen/homebridge-roomba-stv#65]
Due to the layout of my home, I typically clean one or a subset of rooms at a time, and recently Roomba has added "Clean Zones" that you can target as well.

It'd be cool if rooms and zones could appear as individual switches to clean, allowing us to either clean the whole home or one specific zone.

**Comment by epheterson** [comment]
Not sure how tricky it is to list the rooms and dynamically add switches, but at least for rooms this seems possible with the node API: https://github.com/koalazak/dorita980#cleanRoom

Looks like "Clean Zones" aren't supported yet, but it's also in beta so may take some time.

**Comment by Pblondin123** [comment]
Agreed.   The coolest feature i would see for now from what's suggested out there.  As a 2nd suggestion maybe make the rooms parameters in the HB config settings.

**Comment by wallacepost** [comment]
Please add Clean specific rooms / zones feature. It take 3 hrs to clean the whole house, without specific room feature then this plugin is bit unrealistic.

**Comment by whreams** [comment]
+1

In order to remove the opportunity to use the app this is a valid feature for the newer models (s7, s9).
From my perspective, I only use the app for instructing the Roomba to clean specific rooms at certain times or situations during the day.
